### DISCUSSION TITLE
Grid View breaking the Container size and elements inside are flowing out of it on page resizing.

### DISCUSSION THREAD

**Discussion by Alcromido** [flet-dev/flet#800] [Q&A]
Hi, I am learning about flet. Recently I had to use grid view to show multiple things on the screen in a container and whenever I resize the window grid view stretches or compresses my container which leads to overflowing of some of my components. I tried using Row to replace it with the parameter wrap = True but I was not able to add scrolling to the Row. So the only solution for smooth scrolling for large number of containers seems to be Grid View. Does someone know how I can make the contents not overflow when resizing the page ?

**Comment by FeodorFitsner** [comment]
Would you share a source code demonstrating the issue?

**Reply by Alcromido** [reply]
Surely.
[grid.txt](https://github.com/flet-dev/flet/files/10349715/grid.txt)

**Reply by ndonkoHenri** [reply]
I wish to test the issue, but the code seems incomplete.
Please build up some complete code with the issue so we/I could easily test.
Thanks.

**Reply by Alcromido** [reply]
[tutorial.zip](https://github.com/flet-dev/flet/files/10365838/tutorial.zip)
I have added both the files with some sample images to use. The main program is the tutori.py and then there is a add products button in the app when run add the 3 images from the zip and then downsize and upsize the `Meesho` window by dragging you will see that elements start to overflow.
